src/frontend/src/components/anchorInput.ts

```diff
@@ -86,21 +92,21 @@ export const mkAnchorInput = (props: {
 
   // Select the input when the component is rendered (and value is empty)
   const selectInput = (elem: Element): void => {
-    if (elem instanceof HTMLInputElement && elem.value === "") {
+    if (elem instanceof HTMLInputElement) {
```

Comment:
if `&& elem.value === ""` has been on purpose deleted, then the comment should probably be updated

Reply:
Oh good catch, thanks